app.js

```diff
@@ -1,10 +1,145 @@
 /* Imports */
-// import { getRandomItem } from './utils.js';
+import { getRandomItem } from './utils.js';
 
 /* State */
+// let shell = ['shell-1', 'shell-2', 'shell-3'];
+let pearl = ['pearl-1', 'pearl-2', 'pearl-3'];
+let guess = ['guess1', 'guess2', 'guess3'];
```

Comment:
it might be a good idea to have these values be ```let guess = ['shell-1', 'shell-2', 'shell-3'];``` (or shell1 etc) so they're consistent with the element they'll be pointing at